app/extensions/brave/locales/en-US/extensions.properties

```diff
@@ -29,3 +29,5 @@ torrent= Torrent Viewer
 torrentDesc=Uses WebTorrent to display torrents directly in the browser. Supports torrent files and magnet links.
 vimium= Vimium
 vimiumDesc=
+honey=Honey
```

Comment:
I think the convo happening regarding extensions name is my fault. I actually created that for extensions such as LastPass where the name is _LastPass: Free Password Manager_ -- which includes product name + tagline. If that's desirable to have the name as-is I +1 on hardcoding names. Sorry for confusion :)